install/install-sh

```diff
@@ -161,106 +190,108 @@ if [ "$sysname" == "Darwin" ]; then
     ##
     echo "found $sysname (Mac OS X), installing joy ..."
 
-    # Create /usr/local/bin directory if it does not currently exist
-    create_directory /usr/local/bin
+    # Create ${PREFIX}/bin directory if it does not currently exist
+    create_directory ${PREFIX}/bin
 
-    # Create /usr/local/var directory if it does not currently exist
-    create_directory /usr/local/var
+    # Create ${PREFIX}/var directory if it does not currently exist
+    create_directory ${PREFIX}/var
 
-    # Create /usr/local/var/joy directory if it does not currently exist
-    create_directory /usr/local/var/joy
+    # Create ${PREFIX}/var/joy directory if it does not currently exist
+    create_directory ${PREFIX}/var/joy
 
-    # Create /usr/local/var/log directory if it does not currently exist
-    create_directory /usr/local/var/log
+    # Create ${PREFIX}/var/log directory if it does not currently exist
+    create_directory ${PREFIX}/var/log
 
-    # Create /usr/local/etc directory if it does not currently exist
-    create_directory /usr/local/etc
+    # Create ${PREFIX}/etc directory if it does not currently exist
+    create_directory ${PREFIX}/etc
 
-    # Create /usr/local/etc/joy directory if it does not currently exist
-    create_directory /usr/local/etc/joy
+    # Create ${PREFIX}/etc/joy directory if it does not currently exist
+    create_directory ${PREFIX}/etc/joy
 
-    # Create /usr/local/share directory if it does not currently exist
-    create_directory /usr/local/share
+    # Create ${PREFIX}/share directory if it does not currently exist
+    create_directory ${PREFIX}/share
 
-    # Create /usr/local/share/man directory if it does not currently exist
-    create_directory /usr/local/share/man
+    # Create ${PREFIX}/share/man directory if it does not currently exist
+    create_directory ${PREFIX}/share/man
 
-    # Create /usr/local/share/man/man1 directory if it does not currently exist
-    create_directory /usr/local/share/man/man1
+    # Create ${PREFIX}/share/man/man1 directory if it does not currently exist
+    create_directory ${PREFIX}/share/man/man1
 
     # Install the binary executable
-    cp ${APP_ROOT}/bin/joy /usr/local/bin
+    cp ${APP_ROOT}/bin/joy ${PREFIX}/bin
     retval=$?
     if [ $retval -ne "0" ]; then
-        echo "error: could not copy bin/joy to /usr/local/bin"
-        exit
+        echo "error: could not copy joy to ${PREFIX}/bin"
+        exit 1
     fi
 
     # Install query
-    cp ${APP_ROOT}/query.py /usr/local/bin/
+    cp ${APP_ROOT}/query.py ${PREFIX}/bin/
```

Comment:
Can you please remove this whole block of code for query.py? It no longer exists.